docs/content/gs-adding-users-proc.adoc

```diff
@@ -1,41 +1,39 @@
 [id='gs-adding-users-proc']
 
-= User Management
+= Managing Users
 
-This section covers how to manage users within the Integreatly environment.
+How to manage users within the {PRODUCT} environment.
 
-== Accessing the SSO Admin Console
-- Login to the OpenShift console as the `customer-admin` user (The credentials for this user were sent with your welcome email).
-- Navigate to the `sso` project.
-- Go to *Applications* > *Routes*
-- Click on the `sso` route hostname to open the Single Sign-On (SSO) landing screen.
-
-image:../assets/images/gs-adding-users-sso-route.png[SSO route]
-
-- Click on the `Administration Console` link on the SSO landing screen.
-- Login to the SSO administration console as the `customer-admin` user.
-
-== Viewing Users in the SSO Admin Console
-- Navigate to the *Users* menu.
-- Click the *View all users* button at the top of the page. This will show the list of users currently available in the Integreatly environment.
+== Accessing the Red Hat Single Sign-On administration console
+. Login to the OpenShift console as the `customer-admin` user. 
+. Navigate to the *sso* project.
+. Retrieve the credentials for the Red Hat Single Sign-On administration console:
+.. Click *Secrets* in the *Resources* menu.
+.. Click *credential-rhsso* in the list of Secrets.
+.. Click *Reveal Secret* in the  *credential-rhsso* detail page.
+.. Note the values for *SSO_ADMIN_USERNAME* and *SSO_ADMIN_PASSWORD*.
+.. Click *Overview* to navigate back to the *sso* project overview page.
```

Comment:
I think the part of getting the route can be removed and replaced by just giving the url to the `openshfit` realm of the SSO admin console `https://<cluster-host>/auth/admin/openshift/console`